README.md

```diff
@@ -4,33 +4,79 @@
-   3. Ввести команду **python venv birdenv**. (birdenv - имя окружения, может быть любым)
-   4. Активировать окружение, использовав команду **birdenv/Scripts/activate**.
-4. В виртуальном окружении установить необходимые библиотеки из файла _model\_inference/requirements.txt_, использовав команду **pip install -r model\_inference/requirements.txt**.
+   3. Создать виртуальное окружение:
+
+      ```bash
+      python venv birdenv
+      ```
+
+      birdenv - имя окружения, может быть любым
+   4. Активировать окружение:
+
+      ```bash
+      birdenv/Scripts/activate
+      ```
+
+4. В виртуальном окружении установить необходимые библиотеки из файла _model\_inference/requirements.txt_, использовав команду:
+
+   ```bash
+   pip install -r model\_inference/requirements.txt
+   ```
 
 Файл с весами и конфигурацией обученной модели _bird_recognition.h5_ будет установлен автоматически при первом использовании bird_predictor.py.
 Датасеты, использованные при обучении и тестировании модели, можно скачать с [Google Drive](https://drive.google.com/drive/folders/1ZDlgOtVB-Jdkqt-cQrEdyl-QMCJwF2kq?usp=sharing).
 
+#### 2 вариант: Используя расширение Dev Containers для VSC
+
+1. Склонировать репозиторий, используя:
+
+   ```bash
+   git clone https://github.com/Lazarevasn/bird-img-recognition.git
+   ```
+
+2. Установить расширение Dev Containers для VSC, если оно не было установлено ранее.
+3. Открыть подкаталог /model_inference в VSC.
+4. Воспользоваться опцией Reopen in Container в меню Open a Remote Window.
+
 ### Запуск
 
 Для запуска модели на инференс необходимо запустить файл _model\_inference/bird_predictor.py_ и передать в качестве параметра путь к изображению на распознавание.
-Для этого, находясь в активированном окружении, необходимо ввести команду:
+Для этого, либо находясь в виртуальном окружении (1 вариант), либо в терминале запущенного Dev Container в VSC (2 вариант), необходимо ввести команду:
 
-**python bird_predictor.py chicken.jpg**
+```bash
+python bird_predictor.py chicken.jpg
+```
 
-Вместо chicken.jpg укажите путь к файлу с изображением.
+Вместо chicken.jpg укажите путь к файлу с изображением. При использовании 2 варианта способа инференса модели требуется, чтобы изображение находилось в каталоге model_inference.
 
 ## Работа с JupyterLab с помощью Docker
 
 ### Подготовка и запуск
 
-1. Скачать и распаковать архив с файлами репозитория в локальный каталог.
+1. Склонировать репозиторий, используя:
+
+   ```bash
+   git clone https://github.com/Lazarevasn/bird-img-recognition.git
+   ```
+
 2. Открыть командную строку или терминал и перейти в подпапку /jnotebook.
-3. Создать образ, используя команду **docker-compose build**.
-4. По завершении процесса создания образа использовать команду **docker-compose up**.
+3. Создать образ, используя:
+
+   ```bash
+   docker-compose build
+   ```
+
+4. По завершении процесса создания образа использовать следующую команду для запуска:
+
+   ```bash
+   docker-compose up
+   ```
```

Comment:
We can also use the command  docker-compose up --build. It's more compact and easier to understand.